canonical-web-and-design/maas-ui
# Add OverrideTestForm.

## Done
* Added the override test form to the global take action menu.

## QA
* Open devtools and monitor websocket requests.
* Put a selection of machines into a failed testing state, ensuring it has script results.
* From the global take action menu, override testing for these machine and choose to suppress tests. 
* Ensure websocket requests are dispatched for the selected machines to both override tests and suppress test results.

## Fixes
Fixes canonical-web-and-design/MAAS-squad#1787
